src/models/room.ts

```diff
@@ -49,6 +49,7 @@ import {
 import { TypedEventEmitter } from "./typed-event-emitter";
 import { ReceiptType } from "../@types/read_receipts";
 import { IStateEventWithRoomId } from "../@types/search";
+import { IAddLiveEventOptions } from "./event-timeline-set";
```

Comment:
Can this please go in the existing event-timeline-set import, it is creating a Sonar smell